### DISCUSSION TITLE
offscreencanvas in publishable lib

### DISCUSSION THREAD

**Discussion by pgg** [vladmandic/human#263] [General]
Hi Vlad,

Cannot find type definition file for 'offscreencanvas' in a new project that uses a package that i build that uses Human
```
[ng] Error: node_modules/@vladmandic/human/types/human.d.ts:1:23 - error TS2688: Cannot find type definition file for 'offscreencanvas'.
[ng] 
[ng] 1 /// <reference types="offscreencanvas" />
[ng]                         ~~~~~~~~~~~~~~~
[ng] 
[ng] 
[ng] Error: node_modules/@vladmandic/human/types/human.d.ts:7:53 - error TS2304: Cannot find name 'OffscreenCanvas'.
[ng] 
[ng] 7 export declare type AnyCanvas = HTMLCanvasElement | OffscreenCanvas;
```

npm i -D @types/offscreencanvas fixes the issue in the new project, but wondering, should human not enforce this instead of installing it manually? Or is there something i should rather do in my lib?

**Comment by vladmandic** [comment]
`@types/offscreencanvas` is explicitly listed in api-extractor `bundledPackages` configuration, so it should be included - and it was in the previous versions, i guess something broke/change in api-extractor functionality that i use to create .d.ts bundle.
i'll take a look.

converting this to an issue so i can track it.

**Reply by vladmandic** [reply]
tracked in #264
